frontend/src/components/auth/auth.module.scss

```diff
@@ -0,0 +1,110 @@
+.formWrapper {
+  display: flex;
+  flex-direction: column;
+  width: 600px;
+  margin-right: 60px;
+  padding: 40px;
+  color: white;
+  background-color: var(--background-gray-400);
+  transform: translateY(-50%);
+}
+
+.main {
+  display: flex;
+  justify-content: right;
+  align-items: center;
+  height: 100vh;
```

Comment:
About mobile devices, there is no mobile design (other than the one for a mobile app), should it be done according to the design of mobile app?

Reply:
no. we will add a mobile version a bit later

Reply:
If we don't use 100vh, .main class will have the height of .formWrapper class, since background is absolutely positioned. In that case, I can't center the .formWrapper component. Is there a way to center the form, without using `height: 100vh` on its parent element?

Reply:
you can use `flex-grow: 1;`